styles.css

```diff
@@ -0,0 +1,231 @@
+@media (max-width: 989px) {
+ article {
+    width: 220px;
+    height: 300px;
+    background-color: #A2C4C9;
+    float: left;
+    border: 3px solid black;
+    border-radius: 5px;    
+    margin: 6px;
+    margin-top: 80px;
+    font-size: x-small;
+    box-sizing: border-box;
+    padding: 10px
+    }
+  #first {
+    height: 300px;
+    width: 400px
+  }
+  #second {
+    height: 190px;
+    width: 310px
+  }
+  #third {
+    height: 170px;
+    position: relative;
+    bottom: 75px;
+    width: 700px;
+    
+  }
+  article h1 {
+    text-align: center;
+    font-size: large;
+    margin-top: -6px;
+  }
+  article h3 {
+    text-align: center;
+    font-size: small;
+    margin-top: -14px
+  }
+  article h2 {
+    font-size: small;
+  }
+  header {
+    height: 75px;
+    border-bottom: 2px solid black;
+    margin-left: 25px;
+    background-color: #B6D7A8;
+    position: fixed;
+    top: 0px;
+    width: 100%
+  }
+  header h1 {
+    font-size: large;
+    text-align: center
+  }
+  header h2 {
+    font-size: small;
+    text-align: center;
+    margin-top: -9px
+  }
+  nav {
+    background-color: #D5A6BD;
+    width: 0px;
+    height: 400px;
+    position: absolute;
+    border: 2px solid black;
+    text-align: center;
+    font-size: small;
+    visibility: hidden
+  }
+  nav p {
+    margin-top: 60px;
+    margin-bottom: -100px;
+    margin-left: 10px
+  }
+  #ad {
+    position: absolute;
+    width: 200px;
+    height: 200px;
+    border: 2px solid black;
+    margin: 100px 18px
+  }
+  #ad img {
+    min-width: 100%;
+    max-height: 100%;
+  }
+    
+  section {
+    width: 740px;
+    float: right;  
+  }
+  body {
+    width: 989px;
+    position: relative;
+    margin: 3px;
+    font-family: "Arial Black";
+  }
+  main {
+    width: 759px;
+    height: 600px;
+    float: left;
+  }
+  footer {
+    float: right;
+    height: 50px;
+    border-top: 1px solid black;
+    margin-right: 50px;
+    background-color: #9FC5F8;
+    font-size: x-small;
+    position: fixed;
+    bottom: 0px;
+    width: 100%
+  }
+  footer p {
+    margin-left: 10px;
+    margin-top: 1px
+  }
+}
+
+
+
+

```

Comment:
You shouldn't have to re-write the entire stylesheet for a new media inquiry. Maybe that's just a SASS thing?